### DISCUSSION TITLE
Trigger suggestion code

### DISCUSSION THREAD

**Discussion by n2ng-dev** [ayamir/nvimdots#548] [Q&A]
Hii

How to trigger suggestion code like in Vscode use Ctrl+Space, Ctrl+I
![Screenshot 2023-03-03 at 18 48 59](https://user-images.githubusercontent.com/12078569/222715691-e7d0c93a-e930-40c1-b465-033a3c5cd286.png)

thanks

**Comment by ayamir** [comment]
Do you means code action or doc? The former is `ga`, the latter is `K`.

**Reply by n2ng-dev** [reply]
no, action code is for fix error code
trigger suggestion like code assist, if we forgot the syntax option that we will use

**Reply by aarnphm** [reply]
by default, nvim-cmp should be able to popup cmp window on current buffer.
